server/routes/forms/incentiveLevelCreateForm.ts

```diff
@@ -9,10 +9,11 @@ export default class IncentiveLevelCreateForm extends Form<IncentiveLevelCreateD
   protected validate(): void {
     if (!this.data.name || this.data.name.length < 1) {
       this.addError('name', 'The level’s name is required')
+    } else {
+      this.data.name.trim()
```

Comment:
As discussed offline I suggested trimming the name in either `server/routes/incentiveLevels.ts` or creating a new function to sanitise the input